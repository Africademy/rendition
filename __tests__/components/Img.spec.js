/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import {
  Img,
  Provider
} from '../../dist'

test('Img renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Img />
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
