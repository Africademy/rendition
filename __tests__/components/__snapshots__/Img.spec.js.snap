// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Img renders correctly 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 bBubtL"
>
  <div
    className="sc-bdVaJa lgVSbN"
  >
    <img
      className="sc-bwzfXH OesNc"
    />
  </div>
</div>
`;
